packages/rolldown/tests/fixture.test.ts

```diff
@@ -27,12 +27,36 @@ function main() {
         throw new Error(`Failed in ${testConfigPath}`, { cause: err })
       }
     })
+
   }
+
+  for (const [testConfigPath, testConfig] of Object.entries(testConfigPaths)) {
+    const dirPath = nodePath.dirname(testConfigPath)
+    const testName = dirPath.replace('./fixtures/', '')
+
+    test.skipIf(testConfig.skipComposingJsPlugin)(`${testName}-composing-js-plugin`, async () => {
+      testConfig.config = testConfig.config ?? {}
+      testConfig.config.experimental = testConfig.config.experimental ?? {}
+      testConfig.config.experimental.enableComposingJsPlugins = testConfig.config.experimental.enableComposingJsPlugins ?? true
+      try {
+        const output = await compileFixture(
+          nodePath.join(import.meta.dirname, dirPath),
+          testConfig,
+        )
+        if (testConfig.afterTest) {
+          await testConfig.afterTest(output)
+        }
+      } catch (err) {
+        throw new Error(`Failed in ${testConfigPath}`, { cause: err })
+      }
+    })
+
+  }
+
 }
 
 async function compileFixture(fixturePath: string, config: TestConfig) {
   let outputOptions: OutputOptions = config.config?.output ?? {}
-  delete config.config?.output
```

Comment:
I don't know what this is used for, but since we ran the test twice, deleting would lead to some tests failing, so I removed it,